index.html

```diff
@@ -3293,5 +3022,176 @@ <h1>
 </pre>
   </section>
 
+  <section class="appendix">
+    <h1>Current Issues</h1>
+
+    <p>
+The list of issues below are under active discussion and are likely to
```

Comment:
Feels a bit weird that we need to commit  references to github issues.

Reply:
Standard practice -- reviewers ask when doing a horizontal review, note that this PR is purely editorial, so I'm just moving stuff around in the spec.